environment.js

```diff
@@ -0,0 +1,4 @@
+export const ENVIRONMENT = {
+	cronFrequency: process.env.PACKAGE_CRON_PATTERN || '* */12 * * * *',
```

Comment:
Op wat slaat `PACKAGE` in de environment variable name? Kan dit niet beter `PUBLICATION` zijn?

Reply:
Environment variables ook documenteren in README. Zie bvb. https://github.com/kanselarij-vlaanderen/themis-export-service#configuration